api/routes/parties.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
Hi @MicahBala, use es6 when writing your code. instead of using require, use import instead. A blog was posted on how to write restful api's using es6, kindly go through that

Reply:
Correction done. I replaced require with es6 imports